File: java/CheckPrime.java
public class CheckPrime {
  public static void main(String[] args) {
    int count = 83647;
  	int i = 1;
  	int num_primes = 0;
  	int num_not_primes = 0;
  	int n = 0;
  	int total = 0;
    int flag = 0;
    System.out.println("Checking prime for the first: "+count+" numbers");
    while(i < count) {
        flag = 0;
        n = i;
        int j = 2;
        while(j <= n/2){
          if(n%j == 0){
            flag = 1;
            break;
          }
          j++;
        }
        i++;
        if (flag == 0) {
          num_primes++;
        } else {
          num_not_primes++;
        }
    }
    total = num_primes + num_not_primes;
  	System.out.println("Done!");
  	System.out.println("Primes: "+ num_primes);
  	System.out.println("Not Primes: "+ num_not_primes);
  	System.out.println("Total: "+ total);
  }
}

File: go/check_prime.go
package main

import (
	"fmt"
)

func main() {
	var count = 83647
	var i = 1
	var num_primes = 0
	var num_not_primes = 0
	var n = 0
	var total = 0
	fmt.Printf("Checking prime for the first: %d numbers\n", count)
	for i < count {
		var flag = 0
		n = i
		var j = 2
		for j <= n/2 {
			if n%j == 0 {
				flag = 1
				break
			}
			j++
		}
		i++
		if flag == 0 {
			num_primes++
		} else {
			num_not_primes++
		}
	}
	total = num_primes + num_not_primes
	fmt.Println("Done!")
	fmt.Println("Primes:", num_primes)
	fmt.Println("Not Primes:", num_not_primes)
	fmt.Println("Total:", total)
}
